backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/VdsCpuUnit.java

```diff
@@ -18,6 +29,24 @@ public VdsCpuUnit(int socket, int core, int cpu) {
         this.socket = socket;
         this.core = core;
         this.cpu = cpu;
+        this.cpuPinningPolicy = CpuPinningPolicy.NONE;
+        this.vmId = new ArrayList<>();
+    }
+
+    public VdsCpuUnit(int socket, int core, int cpu, List<Guid> vmId) {
```

Comment:
I would removed this constructor completely as I think it is not used and it breaks the constraint between vmIds and cpu policy.